upgrade/tests/test_check_constraints_file_path.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+from mock import patch
+from upgrade.scripts.upgrade_python_package import get_constraints_file_path
+from .conftest import VENV_PATH, install_local_package
+
+
+def test_get_constraints_for_dependency1_from_venv_expect_constraints_exist_and_match():
+    full_package_name = "oll_dependency1-2.0.1-py2.py3-none-any.whl"
```

Comment:
Can create a generic test here - wheel and expected as inputs since the 3 tests have the same code basically. Maybe with `parameterize`

Reply:
Makes sense